shared/teams/confirm-modals/confirm-kick-out.tsx

```diff
@@ -51,6 +51,15 @@ const ConfirmKickOut = (props: Props) => {
 
   // TODO(Y2K-1592): do this in one RPC
   const onRemove = () => {
+    dispatch(
+      TeamsGen.createTeamSetMemberSelected({
+        clearAll: true,
+        selected: false,
+        teamID: teamID,
+        username: '',
+      })
+    )
```

Comment:
should also wipe out `createChannelSetMemberSelected`, since we can get here through that path too.

Reply:
I think that might be better for Y2K-1590, where in a separate onRemove members are kicked out of that channel, when that's implemented